contracts/DLCManager.sol

```diff
@@ -258,6 +266,41 @@ contract DLCManager is
         return false;
     }
 
+    /**
+     * @notice  Checks mint eligibility.
+     * @dev     Checks if the amount is non-zero.
+     * @dev     If PoR is disabled, returns true.
+     * @dev     If PoR is enabled, checks if the new total value minted is within bounds.
+     * @dev     If the PoR check fails, reverts with an error.
+     * @param   amount  dlcBTC to mint.
+     * @param   currentTotalMinted  total minted value in all vaults.
+     * @return  bool  whether a call to _mint should happen.
+     */
+    function _checkMint(
+        uint256 amount,
+        uint256 currentTotalMinted
+    ) internal view returns (bool) {
+        if (amount == 0) {
+            return false;
+        }
```

Comment:
bit of an optimization, but is it really worth it? Is this something that we believe will happen often and thus the optimization is worth the extra code?

Reply:
Yes this happens every time we Withdraw. Until this change, every withdraw also emits a Mint event with a value of 0. This change would stop this behaviour and save gas for the Coordinator.